bucket/7zip.json

```diff
@@ -2,7 +2,7 @@
     "homepage": "https://www.7-zip.org/",
     "description": "A multi-format file archiver with high compression ratios",
     "license": {
-        "identifier": "LGPL-2.0-only,BSD-3-Clause",
+        "identifier": "LGPL-2.0-only|BSD-3-Clause",
```

Comment:
`,` is valid.

https://github.com/lukesampson/scoop/blob/b9b48182b4d921aa92e9b3e53ab2d2c376d467d7/libexec/scoop-info.ps1#L72

Reply:
> If multiple licenses, seperate licenses with a comma (,). If dual licensed, seperate licenses with a pipe symbol (|).

Reply:
[Dual license](https://en.m.wikipedia.org/wiki/Multi-licensing) means the end-user can choose which license to use. Simultaneous licenses are when different files are licensed under different licenses.

Reply:
Then maybe we should use `,` in most manifest? And I have another question, do we need to list all license? For some app it's really diffcult. I prefer to list one or two license and add a url. If we have a clear and doable rule in doc it will be much easier. I'll check again later.

Reply:
In my opinion, if there are more than 2 or 3 licenses, we just add `,...` at the end. With one caveat: if they are not all open-source licenses (e.g. cockroachdb) we list all non-open source licenses first.

Reply:
Thanks for your suggestion. I'll apply it later. Could you add it into doc?

Reply:
I added it to the docs at https://github.com/lukesampson/scoop/wiki/App-Manifests#license :

If it is difficult to determine all of the different licenses, it is acceptable to add `,...` at the end of the SPDX list. If there are both open-source and not open-source licenses, please list all non-open source licenses first.

Reply:
Thanks! It's much more clear. I just wonder do we need `Non-licensed` ? Generally if an app has no license it means all right reserved and I prefer to just  tag it as `Freeware` .

Reply:
> I just wonder do we need `Non-licensed` ?

We certainly don’t need it. Perhaps we just use `Unknown` for when software doesn’t appear to have a license. In practice, this is seldom, if ever.

> how to deal with a free license not in SPDX?

We could provide our own identifiers, such as `Unlicense`, defining them in a `licenses.txt` file, such as:
```
Identifer\tDescription\tURL
Unlicense\tThe Unlicense\thttps://unlicense.org/
...

Reply:
I'm affraid I didn't understand what you mean. There are lots of freeware without a license or eula, should I tag them as Non-license or Unknown?